examples/jsm/controls/OrbitControls.js

```diff
@@ -248,7 +250,7 @@ class OrbitControls extends EventDispatcher {
 				offset.applyQuaternion( quatInverse );
 
 				position.copy( scope.target ).add( offset );
-				setPosition( scope.object, position );
+				scope.object.parent.worldToLocal( position );
```

Comment:
Will this break if object has no parent?

Reply:
It always has a scene as a parent I think

Reply:
I think it does not... Use `obj.parent ? obj.parent.worldToLocal( position ) : position.copy9 obj.position )`.

Reply:
When would the camera not be at least parented in a scene?

Reply:
Why don't you just use this?
```js
scope.object.worldToLocal( position );
```

Reply:
> When would the camera not be at least parented in a scene?

In almost all cases the camera is not added to the scene.

Reply:
> Why don't you just use this?
> 
> ```js
> scope.object.worldToLocal( position );
> ```

because then I get this behaviour:

https://user-images.githubusercontent.com/5083203/162046662-078dd9a1-2be5-4fd9-9cbb-3e5fa155469d.mp4